### PR TITLE
concurrency / it-academy / JC-117-22 / Victor Polonnikov / Done tasks of 2 projects

### PR DESCRIPTION
**Pull request by VictPol**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/concurrency/backup_utility/BackupUtility.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                fin.read(buffer, 0, buffer.length);
+                fos.write(buffer, 0, buffer.length);
+            } catch (IOException ex) {
+                System.out.println(ex.getMessage());
+                this.isCopiedByByte = false;
+            }
+        }
+        return copyDir.getAbsolutePath();
+    }
+}
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/concurrency/backup_utility/BackupUtility.java:59]: where is concurrency?

**Review by rakovets — CHANGES_REQUESTED** [review]
Not correct project 'Producer Queue Consumer'

You need create classes:
- Consumer
- Producer
- Container with Queue

Demo class ISN'T source code! move to test!

File: src/main/java/com/rakovets/course/java/core/practice/concurrency/backup_utility/BackupUtility.java (hunk 1 above)
Comment (VictPol) [reply src/main/java/com/rakovets/course/java/core/practice/concurrency/backup_utility/BackupUtility.java]: fixed

**Comment by VictPol** [comment]
All fixed

**Pull request closed without merging**